lib/public_activity/config.rb

```diff
@@ -1,19 +1,15 @@
 require 'singleton'
+require 'active_support/configurable'
 
 module PublicActivity
   # Class used to initialize configuration object.
   class Config
-    attr_accessor :enabled, :table_name
-    attr_reader :orm
+    include ActiveSupport::Configurable
 
-    def initialize
-      @enabled    = true
-      @table_name = "activities"
-      @orm        = :active_record
-    end
+    config_accessor :enabled do true end
+    config_accessor :table_name do 'activities' end
+    config_accessor :orm do :active_record end
 
-    def orm=(orm)
-      @orm = orm.to_sym
```

Comment:
perhaps it should be fixed at the other places, because people will use symbol here naturually